core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator_test.go

```diff
@@ -588,6 +589,74 @@ func TestCoordinator_ReportIsOnchain(t *testing.T) {
 
 }
 
+func TestCoordinator_ConfirmationDelays(t *testing.T) {
+	t.Run("valid output", func(t *testing.T) {
+		expected := [8]uint32{1, 2, 3, 4, 5, 6, 7, 8}
+		ret := [8]*big.Int{}
+		for i, delay := range expected {
+			// assuming little endian
+			ret[i] = big.NewInt(int64(delay))
+		}
+		coordinatorContract := &mocks.VRFBeaconCoordinator{}
+		coordinatorContract.
+			On("GetConfirmationDelays", mock.Anything).
+			Return(ret, nil)
+		defer coordinatorContract.AssertExpectations(t)
+		c := &coordinator{
+			coordinatorContract: coordinatorContract,
+		}
+		confDelays, err := c.ConfirmationDelays(context.TODO())
+		assert.NoError(t, err)
+		assert.ElementsMatch(t, expected[:], confDelays[:])
+	})
+
+	t.Run("invalid output", func(t *testing.T) {
+		coordinatorContract := &mocks.VRFBeaconCoordinator{}
+		coordinatorContract.
+			On("GetConfirmationDelays", mock.Anything).
+			Return([8]*big.Int{}, errors.New("rpc error"))
+		defer coordinatorContract.AssertExpectations(t)
+		c := &coordinator{
+			coordinatorContract: coordinatorContract,
+		}
+		_, err := c.ConfirmationDelays(context.TODO())
+		assert.Error(t, err)
+	})
+}
+
+func TestCoordinator_KeyID(t *testing.T) {
+	t.Run("valid output", func(t *testing.T) {
+		var keyIDBytes [32]byte
+		keyIDBytes[0] = 1
+		expected := dkg.KeyID(keyIDBytes)
+		coordinatorContract := &mocks.VRFBeaconCoordinator{}
+		coordinatorContract.
+			On("SKeyID", mock.Anything).
+			Return(keyIDBytes, nil)
+		defer coordinatorContract.AssertExpectations(t)
+		c := &coordinator{
+			coordinatorContract: coordinatorContract,
+		}
+		keyID, err := c.KeyID(context.TODO())
+		assert.NoError(t, err)
+		assert.ElementsMatch(t, expected[:], keyID[:])
```

Comment:
I believe that this should be `assert.Equal` and not `assert.ElementsMatch` - the latter ignores the order of elements, whereas in this case we care about the order of the bytes in the byte array (different order implies a different key ID).

Reply:
I see. Thanks for catching that. Updated